torch/_inductor/fx_passes/post_grad.py

```diff
@@ -627,44 +630,69 @@ def reinplace_scatters(graph):
     for node in reversed(graph.nodes):
         storage_to_nodes[get_node_storage(node)].append(node)
         if node.target == aten.copy_.default:
-            copy_args_to_copy_nodes[(node.args[0], node.args[1])] = node
+            src = node.args[0]
+            dst = node.args[1]
```

Comment:
hmm isn't this flipped? I think the user code `dst.copy_(src)` will show up in the graph as:
```
torch.ops.aten.copy_.default(dst, src)
```